pbm/bckuputils.go

```diff
@@ -0,0 +1,146 @@
+package pbm
```

Comment:
I may be missing something but is there a reason we should define this library inside the pbm package?  Are there internal (non-Public) pbm functions that are being used?

If not I would prefer we define this function in mongodb-operator since it seems like we can define it there and technically this repo is a fork of a public repo (we forked because of DNS SRV) that we wish to keep close to public.

Reply:
> we wish to keep close to public
Yep, makes sense.
Closing this PR.